packages/udes-ui-validation/src/exampleComponent/materialUiComponent.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import TextField from "../form/TextField";
+import NumberField from "../form/NumberField";
+import FormValidator, { changeHandler } from "../validation";
+
+// CSS hardcoding will not be used in future, just to facilitate and easy demo
```

Comment:
This comments needs to be addressed

Reply:
I can just remove CSS otherwise I'm not sure what we should do? It looks really bad without it.

Reply:
It needs to go anyway really, @udes-ui/core has a decorator method in the stories to add styles to components